pyproject.toml

```diff
@@ -23,7 +23,7 @@ classifiers = [
 # Requirements
 [tool.poetry.dependencies]
 python = "~2.7 || ^3.5"
-poetry-core = "^1.0.0a5"
+poetry-core = { git = "https://github.com/abn/core.git", branch = "file-uri-scheme-sdist" }
```

Comment:
I assume this will be changed once [#22](https://github.com/python-poetry/core/pull/22) is resolved?

Reply:
Yes.